test/fail/ok/pretty-inference.tc.ok

```diff
@@ -90,42 +90,45 @@ because there is no way to satisfy subtyping
    ((((Nat, Bool), (Nat, Bool)), ((Nat, Bool), (Nat, Bool))),
     (((Nat, Bool), (Nat, Bool)), ((Nat, Bool), (Nat, Bool)))))  <: 
     T  (for argument `x`) 
-pretty-inference.mo:42.1-42.7: type error [M0098], cannot apply function of type
-  <T, U>(x : T, u : U) -> Nat
+pretty-inference.mo:42.1-42.11: type error [M0098], cannot apply function of type
+  <T, U>(x : T, u : U, U -> ()) -> ()
```

Comment:
I just added sth that would preserve the long error, because apparently this file tests pretty printing